### DISCUSSION TITLE
What is the process for complaints?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#638] [Q&A]
The GitHub Discussions area has been a good places for people creating and using MRF files to discuss issues they are having.
However, what I don't understand are the processes for getting the issues resolved.

JR-CMS recently posted that complaints about specific files should be submitted using the webform. What is going to happen to these submissions? Is there a team following up on the complaints?  Will they contact the payers? Will the submitters ever be told what has happened?

The webform requires me to submit my name, phone number, email and quite a few other fields. For almost two years there has been a similar form for complaints about hospital price transparency MRFs. I have submitted dozens of complaints using that form and I have never heard from anyone who might be following up on the complaint.  Nor is it clear that anyone has done anything with most of these complaints. The process is not at all transparent.

**Comment by shaselton-usds** [comment]
@nmayle If there is a compliance complaint, then you should follow @JR-CMS [instructions](https://github.com/CMSgov/price-transparency-guide/discussions/635#discussioncomment-4411996). The compliance process is a bit different than the hospital price transparency (HPT) initiative due to states being involved and also the simple fact that HPT and TiC are run by two completely different groups within CMS.

I don't have much insight into the compliance process happening with HPT but I'd imagine that overall when a case is opened it would be meant to be handled in an opaque way until the case is closed/resolved.

Technical recommendations, questions, thoughts that are made here for this schema are evaluated, discussed, and then ultimately incorporated or otherwise. The size of the technical recommendation will be considered if it can be incorporated as well. Smaller, iterative, non-breaking changes are a good target to get a PR accepted under a version 1.x.x. Larger or potentially breaking changes that would make a good candidate for a version 2.0, but would most likely be tied to some regulatory process.

**Reply by JR-CMS** [reply]
In general, CMS does not release information about ongoing investigations or compliance actions. We do review every submission that comes to us.
